client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -21,9 +21,11 @@ class TaxonomyManagerListItem extends Component {
 		translate: PropTypes.func,
 		onClick: PropTypes.func,
 		onDelete: PropTypes.func,
+		isDefault: PropTypes.bool,
```

Comment:
We tend to favor just the adjective for prop names (`compact` instead of `isCompact`) and `is-` for HTML classes only.